src/librustc/hir/map/mod.rs

```diff
@@ -160,6 +161,8 @@ impl Forest {
     }
 }
 
+pub(super) type HirMap<'hir> = [Vec<Option<IndexVec<ItemLocalId, Option<Entry<'hir>>>>>; 2];
```

Comment:
It takes a while to decipher what the structure looks like and why it looks this way - it'd be great if we could add a doc-comment here with an explanation